src/backend/marsha/core/models/file.py

```diff
@@ -114,6 +114,12 @@ class BaseFile(UploadableFileMixin, BaseModel):
         default=0,
     )
 
+    is_public = models.BooleanField(
+        default=False,
+        verbose_name=_("is %(class)s public"),
+        help_text=_("Is the %(class)s publicly accessible?"),
```

Comment:
:warning: this will not translate correctly because this class will always be "document" and "video"... In french and english it works :smile: but in other languages in won't. I thought of a solution like:
```python
class Video(BaseFile):
    FILE_TYPE_NAME=_("document")
    ...
    verbose_name = _(f"Is {FILE_TYPE_NAME:s} public")
    help_text = _(f"Is the {FILE_TYPE_NAME:s} publicly accessible?")
```